nltk/corpus/reader/knbc.py

```diff
@@ -135,30 +155,33 @@ def _knbc_fileids_sort(x):
                             sorted(fileids, key=_knbc_fileids_sort), encoding='euc-jp')
 
     print(knbc.fileids()[:10])
-    print(''.join( knbc.words()[:100] ))
+    print(''.join(knbc.words()[:100]))
 
-    print('\n\n'.join( '%s' % tree for tree in knbc.parsed_sents()[:2] ))
+    print('\n\n'.join(str(tree) for tree in knbc.parsed_sents()[:2]))
 
     knbc.morphs2str = lambda morphs: '/'.join(
-        "%s(%s)"%(m[0], m[1].split(' ')[2]) for m in morphs if m[0] != 'EOS'
-        ).encode('utf-8')
+        "%s(%s)" % (m[0], m[1].split(' ')[2]) for m in morphs if m[0] != 'EOS'
+    ).encode('utf-8')
```

Comment:
It is suspicious we have to encode string to utf8 - shouldn't corpus reader work on unicode? And does the demo work in Python 3?